config/routes.rb

```diff
@@ -244,6 +244,9 @@
     get 'terms' => 'terms#index'
   end
 
+  # Relay
+  get ".well-known/social-relay" => "social_relay#well_known"
```

Comment:
`.well-known` URIs are specified in RFC 5785, so please use something like `x-diaspora-social-relay` or even `x-social-relay` to prevent potential conflicts with new additions. :)